README.md

```diff
@@ -82,6 +83,16 @@ You now have a working setup from which you can build your own telemetry test en
 
 ***
 
+## Docker build
+```text
+# Download the files
+# Update the .yaml manifests
+# build
+jarFile=$(ls -1 *.jar)
+imageName="otel-test-image"
```

Comment:
Wouldn't it be easier to include the `curl` of the fat jar in the docker build? this way it is self contained. As a next step we could think about building it and publishing it via ghcr.io maing it even easier for people to consume it

Reply:
Wanted to keep it simple; then we would also need to think of proxies, Docker-in-Docker situation to be complete right?